src/Reflection.agda

```diff
@@ -421,7 +455,7 @@ mutual
              < arg₁ , arg₂ >
              (i ≟-Arg-info i′ ×-dec a ≟-Pattern a′)
 
-  _≟-Args_ : Decidable (_≡_ {A = List (Arg Term)})
+  _≟-Args_ : Decidable (_≡_ {A = Args Term})
```

Comment:
If you can import `≡-dec` from `Data.List.Properties` then this whole proof can be removed (you may not be able to because of dependency cyles?). Same for `_≟-ArgPatterns_` patterns.

Reply:
I tried, but the termination checker failed to see they are terminating. I might have time to find out why after coming back from holiday.